GitCommands/Remotes/GitRemoteManager.cs

```diff
@@ -274,8 +274,26 @@ public GitRemoteSaveResult SaveRemote(GitRemote remote, string remoteName, strin
             bool remoteDisabled = false;
             if (creatingNew)
             {
-                output = module.AddRemote(remoteName, remoteUrl);
-                updateRemoteRequired = true;
+                if (GetEnabledRemoteNames().FirstOrDefault(r => r == remoteName) != null)
```

Comment:
Technically, this check to see if an enabled remote of the same name already exists isn't necessary as the module.AddRemote below will fail and return an error message. But this message is more user friendly. Module.AddRemote returns something like "fatal: remote <name> already exists".